linkerd/core/src/test/scala/io/buoyant/linkerd/RouterTest.scala

```diff
@@ -96,4 +96,21 @@ servers:
     val DstBindingFactory.Namer(interpreter) = router.params[DstBindingFactory.Namer]
     assert(interpreter.isInstanceOf[TestInterpreter])
   }
+
+  test("with retries") {
+    val yaml =
+      """|protocol: plain
+         |client:
+         |  retries:
+         |    backoff:
+         |      kind: jittered
+         |      minMs: 100
+         |      maxMs: 10000
+         |    budget:
+         |      ttlSecs: 30
+         |      minRetriesPerSec: 3
+         |      percentCanRetry: 0.33
+         |""".stripMargin
+    assert(parse(yaml, Stack.Params.empty) != null)
```

Comment:
we should check the value of the `Retries.Budget` stack param in the parsed router.